libs/utils/src/postgres_backend.rs

```diff
@@ -45,6 +45,10 @@ pub trait Handler {
     fn check_auth_jwt(&mut self, _pgb: &mut PostgresBackend, _jwt_response: &[u8]) -> Result<()> {
         bail!("JWT auth failed")
     }
+
+    fn is_shutdown_requested(&self) -> bool {
+        PGBACKEND_SHUTDOWN_REQUESTED.load(Ordering::Relaxed)
```

Comment:
Do we need PGBACKEND_SHUTDOWN_REQUESTED at all anymore?

Reply:
Currently, there are two more implementations of Handler aside from the one in pageserver. One in safekeeper and one in proxy (on the management size). But none of them actually calls to `set_pgbackend_shutdown_requested`. So maybe default implementation can just return false?